features/step_definitions/api_request.rb

```diff
@@ -22,6 +22,7 @@
 end
 
 Given("A submission date of {string}") do |date|
+  travel_to Date.parse(date)
```

Comment:
This needs to used in a block - otherwise it will probably pollute the rest of the cucumber tests